app/workers/check_for_new_commits_on_github_worker.rb

```diff
@@ -43,14 +51,22 @@ def check_for_new_commits
       # All the shas for builds we have run and dump it to a set so we can filter with it
       local_build_shas_set = builds.map(&:sha).to_set
 
-      # Get all commits from the open PRs
-      open_pull_requests = github_service.open_pull_requests(repo_full_name: repo_full_name)
+      # Get all branches for all commit triggers
+      branches = project.find_triggers_of_type(trigger_type: :commit).map(&:branch)
+
+      # Get all commits from the open PRs given a set of branches
+      open_pull_requests = github_service.open_pull_requests(
```

Comment:
do we still need this, if the commit trigger is only aimed to be triggered on new commits on the target branch?